lib/question.js

```diff
@@ -1118,7 +1118,7 @@ module.exports = {
         }
 
         if (variant.workspace_id) {
-          urls.workspaceUrl = `${urlPrefix}/workspace/${variant.workspace_id}`;
+            urls.workspaceUrl = `/workspace/${variant.workspace_id}`;
```

Comment:
```suggestion
            urls.workspaceUrl = `/pl/workspace/${variant.workspace_id}`;
```
Not sure if it should also have `/pl`?